recommend/recommend.go

```diff
@@ -39,6 +44,106 @@ type Deployment struct {
 	Images    []string
 }
 
+// ======================== //
+// == Knox System Policy == //
+// ======================== //
+
+// KnoxFromSource Structure
+type KnoxFromSource struct {
```

Comment:
can we import this from kubearmor and use? rather than redefining them?

Reply:
What are the modifications exactly? Can we maybe upstream it, I don't think we should duplicate this at all unless there's an absolutely compelling reason.  Thanks

Reply:
Yes. I will remove this type and add it from the kubearmor. Kubearmor has the type structure that's need for this. (From)[https://github.com/kubearmor/KubeArmor/blob/main/pkg/KubeArmorController/api/security.kubearmor.com/v1/kubearmorhostpolicy_types.go] here.